modules/broker-core/src/main/java/io/ballerina/messaging/broker/core/rest/LoggersApiDelegate.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+package io.ballerina.messaging.broker.core.rest;
+
+import io.ballerina.messaging.broker.auth.AuthException;
+import io.ballerina.messaging.broker.auth.authorization.AuthorizationHandler;
+import io.ballerina.messaging.broker.auth.authorization.Authorizer;
+import io.ballerina.messaging.broker.auth.authorization.enums.ResourceAuthScope;
+import io.ballerina.messaging.broker.core.rest.model.LoggerMetadata;
+import io.ballerina.messaging.broker.core.rest.model.ResponseMessage;
+import org.apache.log4j.Level;
+import org.apache.log4j.LogManager;
+import org.apache.log4j.Logger;
+
+import java.util.ArrayList;
+import java.util.Enumeration;
+import java.util.List;
+import java.util.regex.Pattern;
+import javax.security.auth.Subject;
+import javax.ws.rs.BadRequestException;
+import javax.ws.rs.NotAuthorizedException;
+import javax.ws.rs.NotFoundException;
+import javax.ws.rs.core.Response;
+
+/**
+ * Delegate class that handles Loggers REST API requests.
+ */
+public class LoggersApiDelegate {
+
+    public static final String LOGGERS_API_PATH = "/loggers";
```

Comment:
It seems this is only used by the test classes. Shall we remove the constant from this class?